srcs/app/http/response/HttErrorResponse.cpp

```diff
@@ -0,0 +1,18 @@
+#include "HttpErrorResponse.hpp"
```

Comment:
This filename is missing the 'p' after the Htt.

HttResponseFactory.cpp -> HttpResponseFactory.cpp